src/internal/m365/collection/site/collection_test.go

```diff
@@ -282,65 +279,3 @@ func (suite *SharePointCollectionSuite) TestCollection_streamItems() {
 		})
 	}
 }
-
-// TestRestoreListCollection verifies Graph Restore API for the List Collection
```

Comment:
please separate code movement and code changes into different PRs for ease of review.